db/keyval/etcdv3/plugin_impl.go

```diff
@@ -77,5 +86,19 @@ func (p *Plugin) Init() error {
 		},
 	)
 	p.Skeleton = skeleton
-	return p.Skeleton.Init()
+	err = p.Skeleton.Init()
+	if err != nil {
+		return err
+	}
+
+	// register for providing status reports (polling mode)
+	p.StatusCheck.Register(PluginID, func() (statuscheck.PluginState, error) {
```

Comment:
I am wondering If this might be optional, probably nil check would be good.

Anyway, please update [README](https://github.com/ligato/cn-infra/blob/master/db/keyval/etcdv3/README.md) and [inject the dependency](https://github.com/ligato/cn-infra/blob/master/examples/simple-agent/generic/generic.go)